### ISSUE TITLE
Cannot work within iframe & Dynamic content through extension or AJAX

### ISSUE THREAD

**Issue by arif-so** [artemave/translate_onhover#110]
Your extension really helps me a lot in my daily life. I want to inform you about one small issue and request to fix it. Imagine I am using a third-party extension called 'Slider AI'. This extension creates dynamic popups when I ask it any question. Inside this newly created dynamic content, TranOver does not work. It also doesn't work inside iframes.

If you need more information, please let me know. I am ready to share.

**Comment by artemave** [comment]
Well it's hard to know why slider ai doesn't work, but iframes are supposed to be working. Could you give give an example?

**Comment by arif-so** [comment]
![Screenshot_1](https://github.com/artemave/translate_onhover/assets/127045060/b3c4c7e1-5641-478b-9dcc-7d6c58ea9b11)
If you need more I will give you.

**Comment by arif-so** [comment]
Here are some simple tips you can easily apply: use JavaScript MutationObserver to observe dynamic changes in the entire page (DOM), then initialize the extension after any new change.

**Comment by artemave** [comment]
Shadow dom. I am not sure external scripts can "penetrate" it. But I might be wrong.

> Here are some simple tips you can easily apply: use JavaScript MutationObserver to observe dynamic changes in the entire page (DOM), then initialize the extension after any new change.

Sounds like you know what to do. I'd welcome your pr.

**Comment by arif-so** [comment]
You're correct, brother. I know the solution to the problem, but the issue is I've never worked on extensions before. So, I'm unable to contribute to this project.

**Comment by arif-so** [comment]
You can provide me with the steps for the task, and in the future, I'll attempt to solve it.
